### DISCUSSION TITLE
I changed "default model folder and the model names as well" but still try to download default models

### DISCUSSION THREAD

**Discussion by fi-de** [lllyasviel/Fooocus#2700] [Q&A] [answered]
Hi guys,
I changed "default model folder and the model names as well" in config.txt file.

Which looks like this :

{
  "path_checkpoints": [
    "D:\\AI\\Main_SD_Model_Folder\\stable-diffusion"
  ],
  "path_loras": [
    "D:\\AI\\Main_SD_Model_Folder\\lora"
  ],
  "path_embeddings": "D:\\AI\\Main_SD_Model_Folder\\embeddings",
  "path_vae_approx": "D:\\AI\\Main_SD_Model_Folder\\vae-approx",
  "path_upscale_models": "D:\\AI\\Main_SD_Model_Folder\\gfpgan",
  "path_inpaint": "D:\\AI\\Main_SD_Model_Folder\\inpaint",
  "path_controlnet": "D:\\AI\\Main_SD_Model_Folder\\controlNet",
  "path_clip_vision": "D:\\AI\\Main_SD_Model_Folder\\clip_vision",
  "path_fooocus_expansion": "D:\\AI\\Main_SD_Model_Folder\\expansion",
  "path_wildcards": "D:\\AI\\Main_SD_Model_Folder\\wildcards",
  "path_outputs": "D:\\AI\\AI_Outputs\\Fooocus_Output",
  "temp_path": "C:\\Users\\Lewona\\AppData\\Local\\Temp\\fooocus",
  "temp_path_cleanup_on_launch": true,
  "default_model": "D:\\AI\\Main_SD_Model_Folder\\stable-diffusion\\SDXL-juggernautXL_v8Rundiffusion.safetensors",
  "previous_default_models": [
    "SDXL-juggernautXL_v8Rundiffusion.safetensors",
    "juggernautXL_version8Rundiffusion.safetensors",
    "juggernautXL_version7Rundiffusion.safetensors",
    "juggernautXL_v7Rundiffusion.safetensors",
    "juggernautXL_version6Rundiffusion.safetensors",
    "juggernautXL_v6Rundiffusion.safetensors"
  ]
}

Could you tell me which files and code lines should I change?

**Comment by Rogala** [comment]
Double backslash in path `\\`

**Reply by fi-de** [reply]
Thank you for answer, but when i try to paste the code with backslash \\ here, it does not accept it. 
Here is the screenshot.
![image](https://github.com/lllyasviel/Fooocus/assets/141663269/5e510af7-66e6-48e4-872f-e0c6b479e039)

**Reply by Rogala** [reply, marked as answer]
I don't fully understand you, but I assume that you have the path to the new folders in the **config.txt file**. But in these folders there are no standard juggernautXL_v8Rundiffusion.safetensors and sd_xl_offset_example-lora_1.0.safetensors, and by default, if you look at the file **config_modification_tutorial.txt** if they are not there, you need to load.

Part of the code in **config_modification_tutorial.txt**:
```
"checkpoint_downloads": {
        "juggernautXL_v8Rundiffusion.safetensors": "https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/juggernautXL_v8Rundiffusion.safetensors"
    },
    }, "lora_downloads": {
        "sd_xl_offset_example-lora_1.0.safetensors": "https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/resolve/main/sd_xl_offset_example-lora_1.0.safetensors"
    },
```
should be changed to **config.txt file**:
```
"checkpoint_downloads": {},
"lora_downloads": {},
```
But that's not all, when you change the preset, each **Fooocus\presets\*.json** file in the folder also downloads its **checkpoint and lora** if they are not in the folders. These files need to be corrected too.

**Reply by fi-de** [reply]
I added those **"checkpoint_downloads": {},
"lora_downloads": {},"** to the config.txt and changed the model names in .json presets as well, it worked, thanks :)
